appcheck/firebase-appcheck-debug/src/main/java/com/google/firebase/appcheck/debug/FirebaseAppCheckDebugRegistrar.java

```diff
@@ -29,10 +29,11 @@
  */
 @KeepForSdk
 public class FirebaseAppCheckDebugRegistrar implements ComponentRegistrar {
+  public static final String LIBRARY_NAME = "fire-app-check-debug";
```

Comment:
nit, here and throughout
```suggestion
  private static final String LIBRARY_NAME = "fire-app-check-debug";
```

Reply:
Done.

Reply:
Changed all constants to be private. Can you help me take another look?